core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -2115,10 +2116,6 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
                   samDAO.getPolicySyncStatus(SamResourceTypeNames.billingProject, workspaceRequest.namespace, SamBillingProjectPolicyNames.owner, userInfo).map(_.email)))
               resource <- createWorkspaceResourceInSam(workspaceId, billingProjectOwnerPolicyEmail, workspaceRequest, parentSpan)
               policyEmailsByName: Map[SamResourcePolicyName, WorkbenchEmail] = resource.accessPolicies.map(x => SamResourcePolicyName(x.id.accessPolicyName) -> WorkbenchEmail(x.email)).toMap
-              (googleProjectId, googleProjectNumber) <- traceDBIOWithParent("setupGoogleProject", parentSpan)(_ => DBIO.from(
-                  setupGoogleProject(billingProject, billingAccount, workspaceId, workspaceName, policyEmailsByName, billingProjectOwnerPolicyEmail, parentSpan)))
-              savedWorkspace <- traceDBIOWithParent("saveNewWorkspace", parentSpan)(span =>
-                  createWorkspaceInDatabase(workspaceId, workspaceRequest, bucketName, billingProjectOwnerPolicyEmail, googleProjectId, Option(googleProjectNumber), Option(billingAccount), dataAccess, span))
```

Comment:
moved down.

no policy exists yet when we run setupGoogleProject -- change order of operations so we create the policy earlier

```

[ERROR] [17:50:20.461] [rawls-akka.actor.default-dispatcher-15] o.b.d.r.webservice.RawlsApiService$ - 400 Bad Request
POST https://cloudresourcemanager.googleapis.com/v1/projects/terra-test-1ce34440:setIamPolicy
{
  "code" : 400,
  "errors" : [ {
    "domain" : "global",
    "message" : "Group policy-c1aa4e63-d82c-4598-91a3-60d8f71a6c3e@dev.test.firecloud.org does not exist.",
    "reason" : "badRequest"
  } ],
  "message" : "Group policy-c1aa4e63-d82c-4598-91a3-60d8f71a6c3e@dev.test.firecloud.org does not exist.",
  "status" : "INVALID_ARGUMENT"
}
[ERROR] [04/08/2021 17:50:20.464] [rawls-akka.actor.default-dispatcher-15] [akka.actor.ActorSystemImpl(rawls)] Error during processing of request: 'java.lang.NullPointerException (No error message supplied)'. Completing with 500 Internal Server Error response. To change default exception handling behavior, provide a custom ExceptionHandler.
java.lang.NullPointerException

``